src/Microsoft.IdentityModel.Tokens/Validation/ValidationFailureType.cs

```diff
@@ -134,5 +134,10 @@ private class XmlValidationFailure : ValidationFailureType { internal XmlValidat
         /// </summary>
         public static readonly ValidationFailureType IssuerValidatorThrew = new IssuerValidatorFailure("IssuerValidatorThrew");
         private class IssuerValidatorFailure : ValidationFailureType { internal IssuerValidatorFailure(string name) : base(name) { } }
+
+        /// <summary>
+        /// Defines a type that represents that the issuer signing key validation delegate threw an exception.
+        /// </summary>
+        public static readonly ValidationFailureType IssuerSigningKeyValidatorThrew = new IssuerSigningKeyValidationFailure("IssuerSigningKeyValidatorThrew");
```

Comment:
`IssuerSigningKeyValidatorThrew` -> `IssuerSigningKeyValidatorFailure` ?

Reply:
I am not sure I follow the question here. Is it about the inheritance?

Reply:
+1 to naming. It wasn't immediately clear to me what this var represented. `Failure` instead of `Threw` makes it more clear, I think.

Reply:
We will be reviewing all the `ValidationFailureType` static instances to make sure the names correctly reflect the reason the validation has failed.
We initially had `AudienceValidationFailed`, `LifetimeValidationFailed`, but we are looking into being more specific.

I think this one may need a review on the name to say something like "the user provided issuer signing key validation delegate threw an exception".
Happy to take any ideas for names, otherwise I'll make sure to run the list of names by JM and the larger group when we're happy.